pom.xml

```diff
@@ -93,6 +93,7 @@
     <maven.compiler.source>${jflex.jdk.version}</maven.compiler.source>
     <maven.compiler.target>${jflex.jdk.version}</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.build.outputTimestamp>1</project.build.outputTimestamp>
```

Comment:
I don't know if this is a local time or in a timezone. In any case, an ISO_OFFSET_DATE_TIME format would be easier to read and guaranteed to work with timezone differences such as different machines or DST. Can you please change that.

Reply:
if you want, you can define a meaningful timestamp with something like `2019-10-02T08:04:00Z`
if you use maven-release-plugin, the value will be updated automatically when doing `release:prepare`
if you don't use maven-release-plugin, it will be up to you to update the value or not